src/components/footer.js

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import "./footerStyles.scss";
+
+export const Footer = () => {
+  return (
+    <div className="container">
+      <img className="logo" alt="logo" />
+      <section className="column">
+        <h1 className="heading">O nas</h1>
```

Comment:
There should be only one `<h1>` on a page. I'm not sure why using it in `footer` when from the routes I can say it will appear on any site. Would there be no other `<h1>` tags for sure on them?